snap/snapcraft.yaml

```diff
@@ -0,0 +1,22 @@
+name: gmailctl
```

Comment:
Is it mandatory to use the snap/ directory? If not I would prefer to move this file to the hack/ directory.

Reply:
Unfortunately, I think it is. You can check this [issue](https://github.com/canonical-web-and-design/build.snapcraft.io/issues/1094), the repo was archived but I think `/snap/snapcraft.yaml`, `/snapcraft.yaml` or `/.snapcraft.yaml` are still the only options.

Reply:
Gotcha, thanks.